views/layouts/main.hbs

```diff
@@ -1,175 +1,175 @@
 <!DOCTYPE html>
 <html lang='en'>
 
-<head>
-	{{> head title=title }}
-</head>
+	<head>
+		{{> head title=title }}
+		<script>
+			const BASE_API = "{{getConfigValue 'config/server' 'apiBase' ''}}";
+		</script>
+	</head>
 
-<body>
-	<main>
-		<nav class="navbar d-lg-none navbar-dark bg-default">
-			<div class="container-fluid">
-				<a class="navbar-brand" href="#">{{getConfigValue 'config/site' 'title' 'Org' }}</a>
-				<button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#sidebar" aria-controls="sidebar" aria-expanded="false" aria-label="Toggle navigation">
-					<span class="navbar-toggler-icon"></span>
-				</button>
-			</div>
-		</nav>
+	<body>
```

Comment:
I'm not a fan of the indentation level here personally, I tend to throw HTML and HEAD/BODY at the same level just to pull everything into the margin. But that's a personal choice - feel free to completely ignore me!

Reply:
If I'm honest, I hadn't actually changed anything here but think I got VS to format for me... will be a ballache to have to constantly fight against auto-tools so I'm going to leave this for now!